client/src/localization/languages/Eng.tsx

```diff
@@ -322,6 +322,7 @@ export default {
   com_nav_welcome_assistant: 'Please Select an Assistant',
   com_nav_welcome_message: 'How can I help you today?',
   com_nav_auto_scroll: 'Auto-scroll to Newest on Open',
+  com_nav_enter_to_send: 'Do Send Message with Enter Key?',
```

Comment:
change to "Send messages with Enter Key"

Reply:
updated 2832434ba157810b771043d729b5ff60346ce278